docs/source/examples/Hyperband-synthetic-example.ipynb

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
The example should be cleaned up to make it more interesting to a broad set of people.  Currently it seems to be written to show off details of the Hyperband algorithm itself, rather than as a copy-paste-edit example for naive users.  I suggest a few edits

1.  Add simple prose explaining the situation in which the this applies
2.  Use a real estimator like SGD rather than ConstantFunction
3.  Avoid creating too many threads/processes.  Otherwise users who don't understand what's going on (which is almost all) will do this and be confused why things are slow in practice.
4.  Remove all comments
5.  Remove the use of info
6.  Remove the use of altair, unless we expect most users to have this or it's particularly important

Reply:
Thanks. This was also a working copy I used to debug, which is why all that was present.